.changes/next-release/Feature-5baff9c0-54b5-4ed3-8a3e-a74800cc5af5.json

```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Feature",
+	"description": "CodeWhisperer: Improve Java suggestion quality with enhanced file context fetching"
```

Comment:
The PR title mentions js, ts jsx, tsx. Do we need a changelog for those too?

Reply:
yea that make sense... let me discuss this with the team Justin